### ISSUE TITLE
Audio applet: HDMI audio through dock

### ISSUE THREAD

**Issue by smalltimer** [budgie-desktop/budgie-desktop#779]
Hi,

I use my laptop through a dock which is then connected via HDMI to a monitor+speakers. I have found that the audio applet does not allow switching from laptop audio to HDMI audio. The only way to switch is through the system settings.

Interestingly, once one switches to HDMI audio through the system settings, it is possible to use the applet to switch back to the laptop speakers. But once again, after this switch, it is impossible to revert to HDMI from laptop audio.

TLDR: Changing from laptop audio to HDMI audio not possible through the applet, but vice-versa is.

**Comment by garnieretienne** [comment]
Same issue here with a mini display port to HDMI adaptor. Works fine through the system settings.

**Comment by agoe1179** [comment]
Also experiencing this issue. Sound switch works in the settings but not the applet
